adafruit_motor/stepper.py

```diff
@@ -57,9 +57,38 @@
 """Step a fraction of a step by partially activating two neighboring coils. Step size is determined
    by ``microsteps`` constructor argument."""
 
+_SINGLE_STEPS = (
```

Comment:
wanna make these bytearrays so they take less space? we have to cram this into crikit build for cpx

Reply:
yah..good idea...